# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1449938537 18000
#      Sat Dec 12 11:42:17 2015 -0500
# Node ID 6774cc464ea074805d042f5010a44df0f9dbdf5b
# Parent  61892a245653c0a09f5cc4697e591c9d81d2f089
* file-io.cc (Ffopen): Fix nargin range check, fixes cset f2cd811f0f9e

diff --git a/libinterp/corefcn/file-io.cc b/libinterp/corefcn/file-io.cc
--- a/libinterp/corefcn/file-io.cc
+++ b/libinterp/corefcn/file-io.cc
@@ -626,17 +626,17 @@ When opening a new file that does not ye
 @end deftypefn")
 {
   octave_value_list retval;
 
   retval(0) = -1.0;
 
   int nargin = args.length ();
 
-  if (nargin < 1 || nargin > 4)
+  if (nargin < 1 || nargin > 3)
     print_usage ();
 
   if (nargin == 1)
     {
       if (args(0).is_string ())
         {
           // If there is only one argument and it is a string but it
           // is not the string "all", we assume it is a file to open
